openedx/openedx-platform
# Aj/tnl 6381 upsell msg

## [TNL-6381](https://openedx.atlassian.net/browse/TNL-6381)

### Description

This is the first of a series of PRs for the https://openedx.atlassian.net/browse/TNL-6381

This PR contains just the changes to add the new banner and set its appearance.

Testing steps: 

To see the banner you must be in a verified course on the Course Info page
Then add the ?upgrade=true parameter to the url

The banner should be displayed.

This branch will serve as a feature branch.  Please review the changes here. I will merge this when all the sub tasks have been reviewed and merged into this branch

### Sandbox
- [x] Build a sandbox for your branch and add a link here

Direct testing link: https://staubina-banner.sandbox.edx.org/courses/course-v1:edX+Test101+course/info?upgrade=true


### Testing
- [x] RTL
- [x] safecommit violation code review process

### Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [x] Code review: @cahrens 
- [x] Code review: @jlajoie
- [x] UX review: @lizcohen
- [x] Product review: @marcotuts 

### Post-review
